code/game/objects/effects/mines.dm

```diff
@@ -243,12 +243,12 @@
 
 /obj/effect/mine/sound
 	name = "honkblaster 1000"
-	var/sound = 'sound/items/bikehorn.ogg'
+	var/sound = 'yogstation/sound/misc/quack.ogg'
 	disarm_time = 60 SECONDS // very long disarm time to expand the annoying factor
 	disarm_product = /obj/item/deployablemine/honk
```

Comment:
Honk mine from the quack sound effect?

Reply:
it's a duck

Reply:
its a duck

Reply:
then why isn't it giving the duck mine back?

Reply:
hes true you know

Reply:
oh wait this one wasnt a duck, fixed it